client/src/App.tsx

```
@@ -892,7 +947,18 @@ export default function App() {
-        <div id="lightbox" className="lightbox" aria-hidden={false} onClick={handleLightboxClick}>
+        <div
+          id="lightbox"
+          className="lightbox"
+          aria-hidden={false}
+          onClick={handleLightboxClick}
+          onPointerDown={handleLightboxPointerDown}
+          onPointerUp={handleLightboxPointerUp}
+          style={{ touchAction: 'pan-y' }}
+        >
+          <div className="lightbox-hud" aria-hidden={true}>
+            FILE: {images[lightboxIndex]?.fileName || ''} // {lightboxIndex + 1} OF {images.length}
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)